aws-globalaccelerator-accelerator/src/main/kotlin/software/amazon/globalaccelerator/accelerator/DeleteHandler.kt

```diff
@@ -23,7 +24,7 @@ class DeleteHandler : BaseHandler<CallbackContext?>() {
         val inferredCallbackContext = callbackContext ?: CallbackContext(HandlerCommons.NUMBER_OF_STATE_POLL_RETRIES)
         val model = request.desiredResourceState
         val foundAccelerator = HandlerCommons.getAccelerator(model.acceleratorArn, proxy, agaClient, logger)
-                ?: return ProgressEvent.defaultSuccessHandler(model)
+                ?: return ProgressEvent.failed(model, callbackContext, HandlerErrorCode.NotFound, "Accelerator Not Found")
```

Comment:
Can you confirm if it matches the other customer facing messages?

Reply:
Yes.  This was patterned after existing messages in the code.